plugin/LanguageClient.vim

```diff
@@ -2,6 +2,11 @@ if has('nvim')
     finish
 endif
 
+if !exists('yarp#py3')
```

Comment:
Should be `*yarf#py3` instead of `yarf#py3`.

Reply:
Sorry, can you explain that one further? I think the formatting is messed in email.

Reply:
Sorry for the typo. I meant yarp.

For chenking function existence in vimscript, you will need to use *funcname, instead of just funcname. Check doc by :h exists

If the formatting is messsed up in email, check it on github

Reply:
Interesting. I wonder how I never ran into that before. Fixed!